streams/copy.ts

```diff
@@ -22,8 +22,7 @@ import type { Reader, Writer } from "../io/types.ts";
  * @param dst The destination to copy to
  * @param options Can be used to tune size of the buffer. Default size is 32kB
  *
- * @deprecated (will be removed after 1.0.0) Use
- * {@linkcode ReadableStream.pipeTo} instead.
+ * @deprecated (will be removed in 0.214.0) Import from {@link https://deno.land/std/io/copy.ts} instead.
  */
 export async function copy(
```

Comment:
Note: a `copySync()` could be implemented. But there's never been any demand for it. I also wonder whether there's demand for `readAllSync()` and `writeAllSync()`.

We could also name this `copyAll()` to align it with `readAll()` and `writeAll()`.